packages/sanes-chrome-extension/src/routes/login/components/LoginForm.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import Button from 'medulas-react-components/lib/components/Button';
+import Block from 'medulas-react-components/lib/components/Block';
+import Form, {
+  useForm,
+} from 'medulas-react-components/lib/components/forms/Form';
+import TextFieldForm from 'medulas-react-components/lib/components/forms/TextFieldForm';
+
+export const ACCOUNT_NAME_FIELD = 'accountNameField';
+export const PASSWORD_FIELD = 'passwordInputField';
+
+interface Props {
+  readonly onLogin: (values: object) => Promise<void>;
+  readonly validate: (values: object) => object;
+}
+
+const LoginForm = ({ onLogin, validate }: Props): JSX.Element => {
+  const { form, handleSubmit, pristine, submitting, invalid } = useForm({
+    onSubmit: onLogin,
+    validate,
+  });
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <Block marginBottom={1}>
+        <TextFieldForm
+          placeholder="Enter your unique identifier"
+          form={form}
+          required
+          fullWidth
+          name={ACCOUNT_NAME_FIELD}
```

Comment:
Remove this Field for now. This field would imply to add extra logic for checking if an account exists after restoring the profile. And also send extra info to the account view status with the initial account to load.

We should leave that for the future. Let's cut the scope and move forward.